### PR TITLE
BaSP-MR-77: Add font color and border color props in button

### PR DESCRIPTION
**Pull request by NicolasKler**
I added font color and border color functionalities to make the button easily used by everyone in the project.

### CODE REVIEW

**Comment by moradpablo** [comment]
it is better if we handle the buttons with the "type". for example to delete, edit and add. depending on which one is used it has a certain color to make it more shared

File: src/Components/Shared/Button/index.jsx [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+        return styles.btnCancel;
+      default:
+        return styles.btn;
+    }
+  };
+
+  return (
+    <div className={styles.btnContainer}>
+      <button className={`${styles.btn} ${getTypeClassName()}`} onClick={props.onClick}>
+        {text}
```
Comment (CavalloFede) [inline src/Components/Shared/Button/index.jsx:28]: i think we need to change  {text} for props.text now that we are using that

**Review by CavalloFede — CHANGES_REQUESTED** [review]

**Review by moradpablo — APPROVED** [review]
seems good , gj!

**Review by leo-balonchard — APPROVED** [review]
Looks good :muscle:

**Pull request merged**